### PR TITLE
updated self-serve

### PR DESCRIPTION
**Pull request by jerae-duffin**
Updated steps to include self-serve.

### CODE REVIEW

File: subscription/upgrade.md [hunk 1]
```diff
@@ -17,28 +17,21 @@ The following section contains information on how to purchase a Docker  subscrip
...
+3. Complete the on-screen instructions.
 
     > **Note**
     >
-    > You can select from the following options: Pro, Team, or Business
-    subscription. The annual subscription is set by default. Select **Pay
-    Monthly**  to change to a monthly subscription.
-
-4. If you are purchasing a Team subscription, provide the organization details and click **Continue to Org Size**. Review your Team subscription and click **Continue to Payment**.
+    >If you have a Business subscription, 5  is the maximum number of users you can add using a credit card and 40 users for purchase orders. If you want to create a purchase order, update the field for the users and click Contact Sales to complete the contact sales form.
```
Comment (amineck) [inline subscription/upgrade.md:32]: nitpick: 5 is the minimum number of users. Also an extra space after 5

File: subscription/upgrade.md [hunk 2]
```diff
@@ -17,28 +17,21 @@ The following section contains information on how to purchase a Docker  subscrip
...
 
-2. Open the drop-down menu next to your username in the top-right corner and select **Billing** to go to the **Billing Details** page. This page displays billing information about your existing user account.
+2. Select your subscription and click **Buy Now**.
 
    > **Note**
    >
-   > Ensure you have selected the correct account on the left side of the
-   **Billing Details** page. For example, if you want to purchase a Docker
-   Pro subscription, you must select your personal account.
+   > If you do not have a subscription, navigate to Organizations and click **Create  Organization**. Proceed with the on screen instructions and click **Continue to Payment**.
```
Comment (amineck) [inline subscription/upgrade.md:26]: I think you mean `if you do not have an organization` 🤔  ? Although the context is a bit confusing, we switched from a personal to a team subscription. 

Maybe we should have 2 sections `purchase a personal subscription` and `purchase a team subscription`

File: subscription/upgrade.md (hunk 1 above)
Comment (jerae-duffin) [reply subscription/upgrade.md:32]: Updated, thanks! @amineck

File: subscription/upgrade.md (hunk 2 above)
Comment (jerae-duffin) [reply subscription/upgrade.md:26]: Thanks! @amineck  I updated organization...Also, for Personal subscription, isn't it free?  The next section below is to upgrade your Pro, Team or Business subscriptions

File: subscription/upgrade.md [hunk 3]
```diff
@@ -47,13 +40,11 @@ To purchase a Docker subscription:
...
-## Purchase Docker Business
-
-The Docker Business subscription is currently available to purchase through the Docker Sales team. To purchase a Docker Business subscription, [Contact Sales](https://www.docker.com/pricing/contact-sales){: target="_blank" rel="noopener" class="_" id="dkr_docs_subscription_btl"}.
-
 ## Upgrade your subscription
 
-To upgrade to a Pro or Team:
+To upgrade from a Personal or Pro subscription to a Team or Business subscription you must first convert your personal namespace to an organization namespace. To learn more, see [convert an account into an organization](../docker-hub/convert-account.md).
+
+To upgrade to a Pro or Team or Business subscription:
```
Comment (usha-mandya) [inline subscription/upgrade.md:47]:
```suggestion
To upgrade to a Pro, Team, or Business subscription:
```

File: subscription/upgrade.md [hunk 4]
```diff
@@ -47,13 +40,11 @@ To purchase a Docker subscription:
...
     **Billing** page.
 
-## Purchase Docker Business
-
-The Docker Business subscription is currently available to purchase through the Docker Sales team. To purchase a Docker Business subscription, [Contact Sales](https://www.docker.com/pricing/contact-sales){: target="_blank" rel="noopener" class="_" id="dkr_docs_subscription_btl"}.
-
 ## Upgrade your subscription
 
-To upgrade to a Pro or Team:
+To upgrade from a Personal or Pro subscription to a Team or Business subscription you must first convert your personal namespace to an organization namespace. To learn more, see [convert an account into an organization](../docker-hub/convert-account.md).
```
Comment (usha-mandya) [inline subscription/upgrade.md:45]:
```suggestion
To upgrade from a Personal or Pro subscription to a Team or Business subscription, you must first convert your personal namespace to an organization namespace. To learn more, see [convert an account into an organization](../docker-hub/convert-account.md).
```

File: subscription/upgrade.md [hunk 5]
```diff
@@ -17,28 +17,21 @@ The following section contains information on how to purchase a Docker  subscrip
...
+3. Complete the on-screen instructions.
 
     > **Note**
     >
-    > You can select from the following options: Pro, Team, or Business
-    subscription. The annual subscription is set by default. Select **Pay
-    Monthly**  to change to a monthly subscription.
-
-4. If you are purchasing a Team subscription, provide the organization details and click **Continue to Org Size**. Review your Team subscription and click **Continue to Payment**.
+    >If you have a Business subscription, 5 is the maximum number of users you can add using a credit card and 40 users for purchase orders. If you want to create a purchase order, update the field for the users and click Contact Sales to complete the contact sales form.
```
Comment (usha-mandya) [inline subscription/upgrade.md:32]:
```suggestion
    > If you have a Business subscription, you can add a maximum of 5 users using a credit card and 40 users for purchase orders. If you want to create a purchase order, update the field for the users and click **Contact Sales** to complete the contact sales form.
```

File: subscription/upgrade.md [hunk 6]
```diff
@@ -9,92 +9,71 @@ redirect_from:
...
-     > **Note**
+     >**Note**
      >
-     > You can also specify the number of users you’d like to add to your organization. On the **Organization size** page, the cost for the annual subscription is set by default. If you prefer to change this, select **Pay Monthly**.
-
-5. You’ll also receive an email confirming your new subscription.
+     > Click **Create Organization** if you want to add another organization to your subscription. Click Buy Now to proceed with the on-screen instructions.
 
-6. Open the drop-down menu next to your username in the top-right corner and select **Billing**.
+     ![Billing personal account](images/Billing-personal-account.png){:width="750px"}
```
Comment (usha-mandya) [inline subscription/upgrade.md:60]: @jerae-duffin The image appears broken. Could you check?

File: subscription/upgrade.md [hunk 7]
```diff
@@ -9,92 +9,71 @@ redirect_from:
...
 
-6. Open the drop-down menu next to your username in the top-right corner and select **Billing**.
+     ![Billing personal account](images/Billing-personal-account.png)
 
-    You can view the information about your new subscription and the total cost
-  on the **Billing** page.
+5. On the **Organization Size** page, select your annual or monthly subscription and update the number of users for your organization. For a Business subscription, the only option available is annually.
 
-## Upgrade to Docker Business
+> **Note**
```
Comment (usha-mandya) [inline subscription/upgrade.md:64]:
```suggestion
> **Note**
>
```

File: subscription/upgrade.md [hunk 8]
```diff
@@ -9,92 +9,71 @@ redirect_from:
...
+> To upgrade to a Business subscription, 5 is the maximum number of users you can add using a credit card and 40 users for purchase orders. If you want to create a purchase order, update the field for the users and click **Contact Sales** to complete the contact sales form.  Then, continue the on-screen instructions for the billing details and click **Purchase**.
 
-To upgrade to a Docker Business subscription, [Contact Sales](https://www.docker.com/pricing/contact-sales){: target="_blank" rel="noopener" class="_" id="dkr_docs_subscription_btl"}.
+6. Click **Continue to Payment** and continue to on-screen instructions and click Purchase to complete your subscription.
```
Comment (usha-mandya) [inline subscription/upgrade.md:67]: This appears at 1. The note above needs indenting . See 
image

**Review by usha-mandya — APPROVED** [review]
LGTM

**Pull request merged**